libs/entry-components/dialog/dialogs/entry-dialog.component.html

```diff
@@ -1,8 +1,10 @@
-<div class="entry-dialog">
+<div class="entry-dialog" role="dialog">
     <div class="dialog-header">
-        <h1 class="title" [ngClass]="{'without-close-icon': hideClose === true}" mat-dialog-title>{{ title }}</h1>
-        <button mat-icon-button *ngIf="!hideClose" type="button" class="close-button" (click)="cancel()">
-            <mat-icon>close</mat-icon>
+        <h1 class="title" tabindex="0" [ngClass]="{'without-close-icon': hideClose === true}" mat-dialog-title>{{ title
```

Comment:
You can replace this class binding with a simpler and more readable syntax since we're only toggling one class. You can omit === true because hideClose will inherently evaluate to true or false based on its value due to JavaScript's truthy and falsy behavior:
`[class.without-close-icon]="hideClose"`

Reply:
I agree